### DISCUSSION TITLE
Efficient way of path prediction

### DISCUSSION THREAD

**Discussion by Pranav400-source** [Tiham2001/tarp#6] [Ideas]
Is there any way to predict any path of any user by less computational complexity and resources required?
If so, please drop some ideas regarding it. It would provide us an edge over other papers.

**Comment by Tiham2001** [comment]
A common way we can try is the use of a graph-based pathfinding algorithm like Dijkstra's algorithm. This algorithm begins with a start node and an "open set" of candidate nodes. At each step, the node in the open set with the lowest distance from the start is examined.

**Reply by Pranav400-source** [reply]
Okay, that is a nice solution.

**Reply by Tiham2001** [reply]
👍

**Reply by shalinimishra31** [reply]
Should work fine.

**Reply by annanya1803** [reply]
Yes, it has considerably low complexity, which is almost linear.
